### DISCUSSION TITLE
Updatable Notify problem

### DISCUSSION THREAD

**Discussion by gentsidis** [quasarframework/quasar#8996] [General - Components / Directives / etc]
Trying the "updatable notifications" example from quasar docs (v2.0.0-beta.12) I get this error:
`Uncaught TypeError: false is not a non-null object`
the error points to the following line, where the code tries to stop the spinner:
```
notif({
  icon: 'done', // we add an icon
  spinner: false, // we reset the spinner setting so the icon can be displayed
  message: 'Uploading done!',
  timeout: 2500 // we will timeout it in 2.5s
})
```
If I remove the `spinner: false` it does not fire the error, but it also does not work as expected.
Does any one encounter the same problem? Is it a bug? Any ideas on how to fix it?

**Comment by metalsadman** [comment]
Please make a codepen of what you're trying to do, since just tested it just now and it works as shown in this example https://next.quasar.dev/quasar-plugins/notify#example--updatable.
